packages/lesswrong/components/recentDiscussion/RecentDiscussionTag.tsx

```diff
@@ -78,7 +78,7 @@ const RecentDiscussionTag = ({ tag, refetch = () => {}, comments, expandAllThrea
   
   const lastVisitedAt = markedAsVisitedAt || tag.lastVisitedAt
   const lastCommentId = comments && comments[0]?._id
-  const nestedComments = useOrderPreservingArray(unflattenComments(comments), (comment) => comment._id);
+  const nestedComments = useOrderPreservingArray(unflattenComments(comments), (comment) => comment.item._id);
```

Comment:
@Will-Howard removing the implicit typing in `useOrderPreservingArray` caused this to error - I think it may have been silently failing to work, since `CommentTreeNode` doesn't have an `_id`, it's a container with an `item` (which is the comment that has an `_id`).  But not sure what functional tests I ought to be trying here

Reply:
Thanks for fixing! I thought this had already been fixed but maybe it got lost in a merge or the PR was closed 🤷‍♂️. The functional test would be that if there are several comments being rendered in the card and you reply to the one at the bottom, it shouldn't jump to the top